### PR TITLE
overview iframe, small changes

### PR DESCRIPTION
**Pull request by iSpeakNerd**
learning github a bit more

### CODE REVIEW

File: docs/downstream-integrations/api/token-gating.md [hunk 1]
```diff
@@ -16,10 +19,10 @@ x-api-key: <api-key>
 
 ## Check roles synchronously
 
-Check roles for an account against a list of token gating rules and get a response from the call for the roles to be assigned or removed.
+Check roles for an account against a list of token gating rules and get a callback response if supplied account passes rule checks Y/N.
 
 ### Trt it out
-
+<!-- Agnes, do we have a "how to use the API explorer section we can link to?-->
```
Comment (iSpeakNerd) [inline docs/downstream-integrations/api/token-gating.md:25]: @agnes512 do we have a link for this?

File: docs/downstream-integrations/api/token-gating.md [hunk 2]
```diff
@@ -3,10 +3,13 @@ id: token-gating
 title: Token Gating
 sidebar_position: 4
 ---
-
-The `/access-control` endpoint allows client applications to interact with Collab.Land's token gating engine to verify a wallet address or crypto account with a list of rules based on the token ownership.
+## Overview
+The `/access-control` [endpoint](link needed) allows client applications to interact with Collab.Land's token gating engine to verify a wallet address or crypto account with a list of rules based on the blockchain token ownership.
```
Comment (iSpeakNerd) [inline docs/downstream-integrations/api/token-gating.md:7]: link needed here

File: docs/downstream-integrations/api/token-gating.md [hunk 3]
```diff
@@ -110,3 +110,14 @@ X-CollabLand-Signature: signature-for-the-request
   ]
 }
 ```
+## Example POC site
```
Comment (iSpeakNerd) [inline docs/downstream-integrations/api/token-gating.md:113]: Adding @agnes512's POC site link

**Review by iSpeakNerd — COMMENTED** [review]
think I did that right?

File: docs/downstream-integrations/api/token-gating.md (hunk 2 above)
Comment (uniyalabhishek) [reply docs/downstream-integrations/api/token-gating.md:7]: @raymondfeng what linking practice should we follow?

Comment (iSpeakNerd) [reply docs/downstream-integrations/api/token-gating.md:7]: @uniyalabhishek he suggested using [relative links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links) I believe 

https://github.com/abridged/collabland-dev/pull/26#discussion_r1028778459

**Pull request closed without merging**